admin/modules/user/users.php

```diff
@@ -1053,6 +1053,11 @@
 
 	if($mybb->settings['enablewarningsystem'] != 0 && $user_permissions['canreceivewarnings'] != 0)
 	{
+		if (!$mybb->settings['maxwarningpoints'])
```

Comment:
There should be no space between `if` and `(` to be consistent with our standards